src/app/pages/storage2/components/disk-info-card/disk-info-card.component.html

```diff
@@ -0,0 +1,17 @@
+<mat-card class="disk-info-card">
+  <mat-card-header>
+    <h3 mat-card-title>{{ 'Disk Info' | translate }}</h3>
+    <button mat-button (click)="onEdit()">{{ 'Edit' | translate }}</button>
+  </mat-card-header>
+  <mat-card-content>
+    <mat-list>
+      <mat-list-item *ngFor="let item of diskInfo">
+        <strong>{{ item.label | translate }}:</strong>
+        <span>{{ item.value | translate }}</span>
```

Comment:
`value` probably doesn't need translation. Values shown need to be exact. Translations are often not exact.